src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -259,16 +258,13 @@ private void generateStatusMessage(List<String> versionToQuery,
             }
             
             ActivityLogEntry activityLogEntry = new ActivityLogEntry(appLog);
-            activityLogEntry = data.filterLogs(activityLogEntry);
             
-            boolean isToShow = activityLogEntry.toShow() && (!activityLogEntry.isTestingData() || data.getIfShowTestData());
+            boolean isToShow = data.filterLogs(activityLogEntry)
+                                && (!activityLogEntry.isTestingData() || data.getIfShowTestData());
```

Comment:
I am not going to refactor `AdminActivityLogPageAction` as there are not sufficient test cases. The test cases are added in #6655. Only after that, I can refactor it.